### DISCUSSION TITLE
想要一个主页屏蔽功能

### DISCUSSION THREAD

**Discussion by HelSing-M** [BewlyBewly/BewlyBewly#191] [General]
有的推广视频比广告还烦人。每次看到首页出现，就没心情逛B站了。能不能出一个此视频不在出现，或者屏蔽关键词，或者屏蔽 同类型标签的功能。

**Comment by hakadao** [comment]
現在用bewlybewly自帶的首頁推介是不會出現這些推廣的，如果使用原版b站首頁的話暫時沒有作處理，預計下版本加個清理功能
